src/main.rs

```diff
@@ -97,11 +97,20 @@ fn main() {
         current_condition["FeelsLikeC"].as_str().unwrap()
     };
     let weather_code = current_condition["weatherCode"].as_str().unwrap();
-    let weather_icon = WEATHER_CODES
-        .iter()
-        .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
-        .map(|(_, symbol)| symbol)
-        .unwrap();
+
+    let weather_icon = if args.nerd {
+        WEATHER_CODES_NERD
+            .iter()
+            .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
+            .map(|(_, symbol)| symbol)
+        
+    } else {
+        WEATHER_CODES
+            .iter()
+            .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
+            .map(|(_, symbol)| symbol)
+    }.unwrap();
+    
```

Comment:
Instead of repeating code, can this be
```
let weather_codes = if args.nerd { WEATHER_CODES_NERD } else { WEATHER_CODES };
let weather_icon = weather_codes
    .iter()
    .find(|(code, _)| *code == weather_code.parse::<i32>().unwrap())
    .map(|(_, symbol)| symbol)
    .unwrap();
    ```
    ?